web3c/signer.js

```diff
@@ -1,3 +1,7 @@
+const nacl = require('tweetnacl');
+const hash = require('js-sha512').sha512_256;
```

Comment:
i am worried about js-sha512 - it doesn't look like it has received much attention, and the code quality has some weirdness - eg if the page sets `window.JS_SHA512_NO_COMMON_JS` it won't work at runtime

Reply:
Do you have an alternative suggested library?

Changing Ekiden's hash function is probably out of scope if we want to release this along with the next Devnet upgrade.

Reply:
I may be flagrantly misunderstanding something, but since we're only using this to recreate the digest and using `tweeetnacl` to do the actual signature verification, I'm not too worried about using this in the short term. Buys us enough time to swap out the hash function in the next version of the key manager.